modules/images-editing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-editing-image-registry-settings-hcp_{context}"]
+= Editing image registry settings for {hcp-title}
+
+After verifying that your image registry is enabled, you can update or edit the image registry with the `rosa edit` command.
```

Comment:
```suggestion
Image registry config can be changed in an existing cluster with the `rosa edit` command.
```

Reply:
Sticking with style guide I have changed it to :
You can change the image registry config with the `rosa edit` command.